package.json

```diff
@@ -19,7 +19,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "alexa-app": "^2.4.0",
+    "alexa-app": "alexa-js/alexa-app",
```

Comment:
Once we've released 2.4.0 I'll change this and make a release of alexa-app-server.